docs/fml-front-end-format.md

```diff
@@ -0,0 +1,198 @@
+---
+id: fml-front-end-format
+title: Feature Manifest Language Front-end Format
+slug: /fml-front-end-format
+---
+
+- Status: new
+- Deciders: @travis79, @jhugman, @teshaq, @k88hudson
+- Date: 2021-10-26
+
+## Context and problem statement
+
+In order to allow product teams to define experimentable application features, there needs to be a manifest file which defines these in a way that is understandable to the Nimbus ecosystem.  The feature manifest should define data types used in the application code to configure features. It should also be able to define a complete default configuration for each application feature.
+
+## Decision drivers
```

Comment:
Another important requirement is that it should be able to replace the existing workflow for feature flags in mobile and desktop applications. i.e. you should be able to tell someone how to change their existing workflow to adopt this one.

Reply:
That makes sense to me 👍